toucan_connectors/mssql/mssql_connector.py

```diff
@@ -18,17 +21,15 @@ class MSSQLConnector(ToucanConnector):
 
     host: str
     user: str
-    db: str = None
     password: str = None
     port: int = None
     connect_timeout: int = None
 
-    @property
-    def connection_params(self):
+    def get_connection_params(self, database):
```

Comment:
General note: shouldn't we define `get_connection_params(database)` (or even `datasource` ?) as the official API and deprecate the `connection_params` property ?

Reply:
Well in most cases you don't need to know the database to connect. But it could be a `get_connection_params(**kwargs)` method indeed

Reply:
Done in https://github.com/ToucanToco/toucan-connectors/pull/96/commits/9b90afcb8e2060c32ad1f55b3f34e7e2bde9d8f2